.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a bug report for the project Amazon FreeRTOS
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**System Information**
+- Hardware board used
+- Operating System [Windows|Linux|MacOS]
+- Version of Amazon FreeRTOS (run ``git describe --tags`` to find it)
+- Demo ran [MQTT|Shadow|Greengrass|OTA|Device Defender|TCP Echo|Sub pub Demo| Simple TCP Server]
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or console output**
+If appropriate, please cut-and-paste the console output to explain the issue
+or add a screenshot.
```

Comment:
We should explicitly ask for a call stack.

Reply:
Call stack would be appropriate with an IDE, although it will be difficult to provide a call stack with ESP32